pandas/_libs/window.pyx

```diff
@@ -1679,18 +1677,18 @@ def roll_generic(object obj,
 # Rolling sum and mean for weighted window
 
 
-def roll_weighted_sum(float64_t[:] values, float64_t[:] weights,
-                      int minp):
+def roll_weighted_sum(float64_t[:] values, float64_t[:] weights, int minp):
+    # `.base` to access underlying ndarray
     return _roll_weighted_sum_mean(values, weights, minp, avg=0)
 
 
-def roll_weighted_mean(float64_t[:] values, float64_t[:] weights,
-                       int minp):
+def roll_weighted_mean(float64_t[:] values, float64_t[:] weights, int minp):
+    # `.base` to access underlying ndarray
     return _roll_weighted_sum_mean(values, weights, minp, avg=1)
 
 
-def _roll_weighted_sum_mean(float64_t[:] values, float64_t[:] weights,
-                            int minp, bint avg):
+cdef _roll_weighted_sum_mean(float64_t[:] values, float64_t[:] weights,
```

Comment:
`cdef void` (unless this returns something)

Reply:
sure.  does this affect perf or is it just more explicit?

Reply:
I'm coming from the explicit angle but I can't imagine it would hurt either

Reply:
actually looks like this should be `cdef ndarray[float64_t]`, will update